automation/check-patch.sh

```diff
@@ -145,9 +145,9 @@ main() {
         "mode=$mode"
         "provider=$provider"
         "inventory_file=$inventory_file"
-        "cluster_type=$cluster_type"
+        "cluster=$cluster"
         "ansible_modules_version=$ansible_modules_version"
-        "openshift_ver=$openshift_ver"
+        "openshift_version=$openshift_version"
```

Comment:
`openshift_version` is also being used in `openshift-ansible`. I think that we shouldn't use it also.

Reply:
Let's keep it so we expose it to the top level